README.md

```diff
@@ -3,45 +3,28 @@
 [![GoDoc](https://godoc.org/github.com/signalfx/signalfx-agent?status.svg)](https://godoc.org/github.com/signalfx/signalfx-agent)
 [![CircleCI](https://circleci.com/gh/signalfx/signalfx-agent.svg?style=shield)](https://circleci.com/gh/signalfx/signalfx-agent)
 
-The SignalFx Smart Agent is a metric agent written in Go for monitoring
```

Comment:
Please don't touch README.md in the root of the repo, that is primarily for developers working on the agent, not customers trying to install it.